### PR TITLE
added some depth to README

### PR DESCRIPTION
**Pull request by Blake-**

### CODE REVIEW

**Comment by mark-cooper** [comment]
Markdown formatting looks a bit off :monkey_face: 

https://github.com/lyrasis/ansible-duplicity-role/blob/readme/README.md

File: README.md [hunk 1]
```diff
@@ -2,15 +2,17 @@ Role Name
 =========
 
 This role installs duplicity from the duplicity source code at http://duplicity.nongnu.org/
-
-Dependencies
-------------
-
-...
+It uses the Duplicity 0.7 Series.
+main.yml installs dependencies using apt. It then uses pop to install python-swift & keystone.
```
Comment (anroots) [inline README.md]: Typo: `pip`, not `pop`.

**Comment by mark-cooper** [comment]
Thanks @anroots 

@Blake- can you update this in the next few days?

File: README.md (hunk 1 above)
Comment (Blake-) [inline README.md]: Thanks for catching that :-)

**Comment by mark-cooper** [comment]
@Blake- :+1: 

One more thing? Can we add an example for each of the more vague variables (duplicity environment, scripts and cron) -- that will be helpful for anyone else who happens to look at this.

**Pull request merged**